src/blots/ImageBlot/index.ts

```diff
@@ -1,8 +1,22 @@
 import Quill from 'quill';
 import { Constants } from '../../utils';
 
-const blotPath = 'blots/block/embed';
-const BlockEmbed = Quill.import(blotPath);
+const blotPath = 'formats/image';
+const BlockEmbed: EmbedBlot = Quill.import(blotPath);
+
+interface EmbedBlot {
+  new(...args: any[]): EmbedBlot;
+  domNode: any;
```

Comment:
@ayush013 Can we have more strict type for domNode?

Reply:
Sure, `HTMLImageElement` will do